config/blade-ui-kit.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+use BladeUIKit\Components;
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Components
+    |--------------------------------------------------------------------------
+    |
+    | Below you reference all components that should be loaded for your app.
+    | By default all components from Blade UI Kit are loaded in. You can
+    | disable or overwrite any component class or alias that you want.
+    |
+    */
+
+    'components' => [
+        'alert' => Components\Alerts\Alert::class,
+        'avatar' => Components\Support\Avatar::class,
+        'carbon' => Components\DateTime\Carbon::class,
+        'checkbox' => Components\Forms\Inputs\Checkbox::class,
+        'color-picker' => Components\Forms\Inputs\ColorPicker::class,
+        'countdown' => Components\DateTime\Countdown::class,
+        'cron' => Components\Support\Cron::class,
+        'dropdown' => Components\Navigation\Dropdown::class,
+        'easy-mde' => Components\Editors\EasyMDE::class,
+        'email' => Components\Forms\Inputs\Email::class,
+        'error' => Components\Forms\Error::class,
+        'form' => Components\Forms\Form::class,
+        'form-button' => Components\Buttons\FormButton::class,
+        'html' => Components\Layouts\Html::class,
+        'input' => Components\Forms\Inputs\Input::class,
+        'label' => Components\Forms\Label::class,
+        'logout' => Components\Buttons\Logout::class,
+        'mapbox' => Components\Maps\Mapbox::class,
+        'markdown' => Components\Markdown\Markdown::class,
+        'password' => Components\Forms\Inputs\Password::class,
+        'pikaday' => Components\Forms\Inputs\Pikaday::class,
+        'social-meta' => Components\Layouts\SocialMeta::class,
+        'textarea' => Components\Forms\Inputs\Textarea::class,
+        'toc' => Components\Markdown\ToC::class,
+        'trix' => Components\Editors\Trix::class,
+        'unsplash' => Components\Support\Unsplash::class,
+    ],
```

Comment:
Let's remove all the components here which we don't use atm. Or comment them out.